src/category_theory/functor_category.lean

```diff
@@ -53,6 +54,13 @@ lemma naturality_app {F G : C ⥤ (D ⥤ E)} (T : F ⟹ G) (Z : D) {X Y : C} (f
   ((F.map f).app Z) ≫ ((T.app Y).app Z) = ((T.app X).app Z) ≫ ((G.map f).app Z) :=
 congr_fun (congr_arg app (T.naturality f)) Z
 
+@[simp] lemma map_vcomp {F : (C ⥤ D) ⥤ E} {G H K : C ⥤ D} {α : G ⟹ H} {β : H ⟹ K} :
+  F.map(α ⊟ β) = F.map α ≫ F.map β :=
```

Comment:
It's a bit awkward, and I'd appreciate advice here.

The problem is that sometimes we write `α : G \nattrans H`, and sometimes we write `α : G \hom H`. If things are written using the `\nattrans` arrow, then you can't compose them using `\gg`, and have to write `vcomp`.

We could introduce a lemma much earlier rewriting `vcomp` into `comp`, and we could really avoid ever writing vcomp in constructing things. What do you think?

Reply:
I haven't thoroughly investigated, but it seems very possible that we could basically deprecate the `\nattrans` arrow in favour of the `\hom` arrow. I'm not sure if this would make this more of less confusing, but it would at least remove the duplication, and `vcomp` would never have to be mentioned.

I'll think about that later. For now... I've just removed the lemma, which is apparently not necessary anymore.

Reply:
I'm in favour of deprecating. I've had quite a lot of situations where I `intro`d a nat-trans as hom in a category, and I would have to `change` it to a `\nattrans` arrow before simplification would kick in.

Reply:
I've made a separate PR #907, that makes the nat_trans arrow a mere synonym for the hom arrow. It's also possible to completely remove it, if people prefer.